docs/introduction/05-realworld-example.mdx

```diff
@@ -1,39 +1,46 @@
 ---
 id: realworld-example
 title: "Getting some real-world data"
-sidebar_label: "Realworld example"
+sidebar_label: "Real-world example"
```

Comment:
idk, i really dont like the `real-world` spelling, this was on purpose, so it matches the URL

```suggestion
sidebar_label: "Realworld example"
```

thoughts @mnmkng?

Reply:
to be honest, I don't really care, but grammarly suggest to use either `real-world` or `real world`, not `realworld`. Also after some googling: [one](https://www.merriam-webster.com/dictionary/real-world#:~:text=%E2%80%9CReal%2Dworld.%E2%80%9D%20Merriam,%2Fdictionary%2Freal%2Dworld.), [two](https://www.oxfordlearnersdictionaries.com/definition/english/real-world).

Anyway - the point is not in the spelling, if you're fine with content changes - I'd rather merge it as currently this page is completely dated and example are not working at all.